src/inject/generic/core/lyrics.js

```diff
@@ -18,7 +20,7 @@ window.addEventListener('load', () => {
     } else {
       clearTimeout(noLyricsTimer);
       const lyricsHTML = lyrics.replace(/\n/g, '<br />');
-      $('#lyrics').html(`<p>${lyricsHTML}</p>`);
+      $('#lyrics').html(`<p scroll="${Settings.get('scrollLyrics', true)}">${lyricsHTML}</p>`);
```

Comment:
Using custom attributes should be `data-scroll=`

Thought why are we using HTML attributes anyway?